FirebaseRTC/public/reviewPage.html

```diff
@@ -47,14 +47,14 @@
                 <div class="col-12">
                     <nav class="main-nav">
                         <!-- ***** Logo Start ***** -->
-                        <a href="/index.html" class="logo">Melly</a>
```

Comment:
Deleted "/" looks sus... maybe check this thing

Reply:
I noticed that this deleted "/" looks inconsistent with the other changes made so far.

Reply:
This issue is fixed in the new commit.